bin/compile

```diff
@@ -93,6 +93,11 @@ fi
 
 speak "Running stack"
 cd "$WORK_DIR"
+# Makefile support (if you need other build tools than Stack that you need to use in conjunction)
+if [ -f $BUILD_DIR/Makefile ]; then
+    speak "Makefile detected"
+    make -C $BUILD_DIR
+fi
```

Comment:
The canonical pattern around make is usually `make; make install` - maybe the behavior here should follow that pattern? If there's a `Makefile`, the expectation is that the buildpack will call `make; make install` on it. That seems reasonable - what do you think?